internal/api/design/package_.go

```diff
@@ -319,6 +322,20 @@ var StoredPackage = ResultType("application/vnd.enduro.stored-package", func() {
 	Required("id", "status", "created_at")
 })
 
+var Page = ResultType("application/vnd.enduro.page", func() {
+	Description("Page represents a subset of search results.")
+	Attribute("limit", Int, "Maximum items per page")
+	Attribute("offset", Int, "Offset from first result to start of page")
+	Attribute("total", Int, "Total result count before paging")
+	Required("limit", "offset", "total")
+})
```

Comment:
This is probably going to be reused, I'd move it to https://github.com/artefactual-sdps/enduro/blob/main/internal/api/design/pagination.go.